en/docs/services/create-your-first-service.md

```diff
@@ -1,176 +1,175 @@
 # Create Your First Service
 
-To learn how to create a service, let's try a simple scenario. In this example, you are creating a service that consumes a public API that fetches the current active COVID-19 cases in a specified country, testing it, deploying it, and then observing its performance.
+To learn how to create a service, let's look at a simple scenario. In this example, you'll create a service that consumes a public API that fetches the current active COVID-19 cases in the United States. Then you'll test it, deploy it, and observe its performance.
 
 ## Step 1: Create the service
 
-To create the service, follow the procedure below: 
+To create the service, follow this procedure: 
 
 1. Access the Choreo Console via https://console.choreo.dev/.
     
 2. Sign in using either your Google or GitHub credentials.
     
-3. In the **Services** page, click **Create**. You are directed to the low-code view.
+3. In the **Services** page, click **Create**. You are directed to **Create Service** page.
 
-4. Under **Create with Choreo**, enter `covid-stats` as the name of your service.
+4. Under **Create with Choreo**, enter **covid-stats** as the name of your service.
 
-5. Select **GET** as the HTTP method, and enter `activecases` as the relative path from host.
+5. Select **GET** as the HTTP method and enter **activecases** as the relative path from host.
 
-    ![Resource Configuration](../assets/img/services/configure-api-trigger.png)
-    
-    Then click **Save API**. Now you have saved the API trigger that starts your service.
+     ![Resource Configuration](../assets/img/services/configure-api-trigger.png)
+
+6. Click **Save API**. 
+
+Now that you have saved the API trigger that starts your service, let's continue with creating the service.
     
-6. Click **API Calls**, and then enter information as follows to configure the HTTP connection that connects the public API used in this scenario to the API trigger you configured.
+Click **API Calls** and then complete the following steps to configure the API trigger to use the HTTP connection that connects the public API used in this scenario to it.
 
-    1. Click **HTTP**.
+1. Click **HTTP**.
```

Comment:
These numbers are not properly indented and sub numbers are not rendered. Check all the other places as well.

![image](https://user-images.githubusercontent.com/16300038/119116807-3e433100-ba46-11eb-8c1d-476010bcbe87.png)

Reply:
It seems we are starting from number one again under the new section. But it is really confusing as the text in between is not appear as a separator for the sections.

Reply:
I think we better have sub steps as sub topics with proper topic under the main sub topic of `Step 1: Create the service`

Reply:
Yeah. This is the numbering related concern I mentioned. It is confusing. So need to see whether we can be flexible with the number of steps under a procedure. Choreo doc team members felt that it is ok to have 10 steps or so under a section.